tap_chromedata/streams.py

```diff
@@ -7,518 +8,327 @@
-    # schema_filepath = SCHEMAS_DIR / "users.json"
-    schema = th.PropertiesList(
-        th.Property("model_year", th.IntegerType),
-        th.Property("division_name", th.StringType),
-        th.Property("model_name", th.StringType),
-        th.Property("hist_style_id", th.IntegerType),
-        th.Property("style_name", th.StringType),
-        th.Property("style_name_wo_trim", th.StringType),
-        th.Property("trim", th.StringType),
-        th.Property("full_style_code", th.StringType),
-        th.Property("style_sequence", th.IntegerType),
-        th.Property("msrp", th.NumberType),
-        th.Property("invoice", th.NumberType),
-        th.Property("destination", th.NumberType),
-        th.Property("model_effective_date", th.StringType),
-        th.Property("model_comment", th.StringType),
-        th.Property("manufacturer_name", th.StringType),
-        th.Property("manufacturer_id", th.IntegerType),
-        th.Property("division_id", th.IntegerType),
-        th.Property("hist_model_id", th.IntegerType),
-        th.Property("market_class", th.StringType),
-        th.Property("market_class_id", th.IntegerType),
-        th.Property("subdivision_name", th.StringType),
-        th.Property("subdivision_id", th.IntegerType),
-        th.Property("style_id", th.IntegerType),
-        th.Property("autobuilder_style_id", th.StringType)
-    ).to_dict()
+"""Convert camelCase words to snake_case"""
+def camel_to_snake(name):
+    name = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
+    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', name).lower()
+    
+"""Base stream class with config parameter getters, tranversers to ACES Mapping data and data cleaning"""
+class ChromeDataStream(Stream):
+    flo=""
     @property
     def url_base(self) -> str:
         """Return the API URL root, configurable via tap settings."""
-        #print(self.config["FTP_URL"],self.config["FTP_USER"],self.config["FTP_PASS"])
         return self.config["FTP_URL"]
     @property
     def url_user(self) -> str:
-        """Return the API URL root, configurable via tap settings."""
-        #print(self.config["FTP_URL"],self.config["FTP_USER"],self.config["FTP_PASS"])
+        """Return the API URL user, configurable via tap settings."""
         return self.config["FTP_USER"]
+    
     @property
     def url_pass(self) -> str:
-        """Return the API URL root, configurable via tap settings."""
-        #print(self.config["FTP_URL"],self.config["FTP_USER"],self.config["FTP_PASS"])
+        """Return the API URL password, configurable via tap settings."""
         return self.config["FTP_PASS"]
-    def get_records(self, context: Optional[dict]) -> Iterable[dict]:
-        def camel_to_snake(name):
-            name = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
-            return re.sub('([a-z0-9])([A-Z])', r'\1_\2', name).lower()
+    """Function for traversing through ACES Mapping folder in the FTP Server"""
+    def reading_ftp(self):
```

Comment:
Typically in python, method descriptions go inside:

```suggestion
    def reading_ftp(self):
      """Function for traversing through ACES Mapping folder in the FTP Server"""
```